dot/network/host.go

```diff
@@ -98,12 +98,15 @@ func newHost(ctx context.Context, cfg *Config) (*host, error) {
 		return nil, err
 	}
 
+	// TODO: What should be the right value of max in and max out?
 	const reservedOnly = false
 	peerCfgSet := peerset.NewConfigSet(
 		uint32(cfg.MaxPeers-cfg.MinPeers),
-		uint32(cfg.MinPeers),
+		uint32(cfg.MaxPeers/2),
```

Comment:
nit it would be nice to set this to some variables before passing them as arguments for readability. For example reading this without my IDE, I have no clue what `uint32(cfg.MaxPeers/2),` is or why it's maxpeers/2. That might apply to `uint32(cfg.MaxPeers-cfg.MinPeers),` as well.

Reply:
added a comment